src/networks.json

```diff
@@ -1350,6 +1350,21 @@
     "logo": "ipfs://bafkreicew2qv3m756m7xs2nrjezinghps7rlessrqjqetvox74w4hov2xe",
     "testnet": true
   },
+  "713715": {
+    "key": "713715",
+    "name": "Sei EVM Devnet",
+    "shortName": "devnet",
+    "chainId": 713715,
+    "network": "testnet",
+    "multicall": "0xcA11bde05977b3631167028862bE2a173976CA11",
+    "rpc": [],
+    "explorer": {
+      "url": "https://seistream.app"
```

Comment:
```suggestion
      "url": "https://seitrace.com"
```

Reply:
There are two explorers for this network, seitrace support all Sei networks 
<img width="322" alt="image" src="https://github.com/snapshot-labs/snapshot.js/assets/15967809/08cdcd1a-4bf6-44b4-bf29-959e0d9369ca">